extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/devconsole/CacheDevConsoleRecorder.java

```diff
@@ -22,15 +22,15 @@
             String message = "";
 
             @Override
-            protected void handlePost(RoutingContext event, MultiMap form) throws Exception {
+            protected void handlePostAsync(RoutingContext event, MultiMap form) {
                 String cacheName = form.get("name");
                 Optional<Cache> cache = CaffeineCacheSupplier.cacheManager().getCache(cacheName);
                 if (cache.isPresent() && cache.get() instanceof CaffeineCache) {
                     CaffeineCacheImpl caffeineCache = (CaffeineCacheImpl) cache.get();
 
                     String action = form.get("action");
                     if (action.equalsIgnoreCase("clearCache")) {
-                        caffeineCache.invalidateAll();
+                        caffeineCache.invalidateAll().await().indefinitely();
```

Comment:
Having that with an `Async` method look weird. Do you really need to make the method `Async`?

Reply:
Yeah, I had raised the same point in another issue

Reply:
If it's not async, then `await().indefinitely()` will cause an exception because of the blocked event loop thread.

Reply:
@mkouba maybe you can help?

Reply:
TBH, I wonder if it's the only thing broken. Because I'm pretty sure we are doing blocking things in a ton of Dev UI actions (reindexing with Hibernate Search for instance).

Reply:
Hm, maybe just register those handlers as blocking by default?

Reply:
In the meantime, is this PR an acceptable fix? The feature is broken right now so it's better to have a temporary fix than nothing. 😃

Reply:
I just gave it a try, this works well for the invalidation:
```
caffeineCache.invalidateAll().subscribe().with(ignored -> {
    event.response().setStatusCode(this.code);
    event.response().end(this.message);
});
```
However, leaving the `actionSuccess` method body empty prevents any other Dev UI request from completing.

Reply:
https://github.com/gwenneg/quarkus/tree/cache-dev-ui-2

My `boolean` idea does not really work, I just tested it (not on that branch).

Reply:
Something like this seems to work just fine: https://github.com/mkouba/quarkus/blob/cache-dev-ui-2/extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/devconsole/CacheDevConsoleRecorder.java#L21-L54

Reply:
That works well indeed, thanks. I updated the PR.

Reply:
@mkouba I let you check the PR and merge if you're happy with it.